.woloxci/config.yml

```diff
@@ -0,0 +1,21 @@
+config:
+  dockerfile: .woloxci/Dockerfile
+  project_name: express-js-bootstrap
+
+services:
+  - postgres
+
+steps:
+  copy_node_modules:
+    - cp -r $NODE_PATH/ ./
+  migrate_db:
+    - node_modules/.bin/sequelize db:migrate
```

Comment:
Who creates the DB?

Reply:
The postgres docker image comes with an already created DB